docs/cart/cart.css

```diff
@@ -0,0 +1,60 @@
+.cart{
+    display: grid;
+    grid-template-columns: 60% 40%;
+    justify-items: center;
+    grid-template-areas: 
+    "cart-items cart-details";
+    position: relative;
+}
+
+.cart-items{    
+    grid-area: cart-items;
+    padding: 3rem;
+    border-right: 2px solid var(--SIDE-NAVBAR-COLOR);
```

Comment:
Better color naming is possible

Reply:
I'm using the same color as side navbar color that is defined in my root. So, should I change the variable name to some generic name or create a new variable but with same color?